packages/office-ui-fabric-react/src/components/Pivot/Pivot.base.tsx

```diff
@@ -206,9 +211,13 @@ export class PivotBase extends React.Component<IPivotProps, IPivotState> {
   };
 
   /**
-   * Renders the current Pivot Item
+   * Renders a Pivot Item
    */
-  private _renderPivotItem(linkCollection: PivotLinkCollection, itemKey: string | undefined): JSX.Element | null {
+  private _renderPivotItem(
+    linkCollection: PivotLinkCollection,
+    itemKey: string | undefined,
+    isActive = true,
```

Comment:
Since it seems like you'll always pass this value to the function, do you have to give it a default?